src/briefcase/windows_icon.py

```diff
@@ -0,0 +1,92 @@
+from ctypes import *
+from win32ctypes.pywin32 import win32api
+
+
+# Structures in ico file
+class IconHeader(Structure):
+    _fields_ = [
+        ("Width", c_byte),
+        ("Height", c_byte),
+        ("Colors", c_byte),
+        ("Reserved", c_byte),
+        ("Planes", c_ushort),
+        ("BitsPerPixel", c_ushort),
+        ("ImageSize", c_int),
+        ("ImageOffset", c_int),
+    ]
+
+
+class GroupIcon(Structure):
+    _fields_ = [
+        ("Reserved", c_ushort),
+        ("ResourceType", c_ushort),
+        ("ImageCount", c_ushort),
+        # ("Enries", PIconDirResEntry),
+    ]
+
+
+# Structure for updating resource
+class IconDirResEntry(Structure):
+    _fields_ = [
+        ("Width", c_byte),
+        ("Height", c_byte),
+        ("Colors", c_byte),
+        ("Reserved", c_byte),
+        ("Planes", c_ushort),
```

Comment:
At column 22: (F405) 'c_byte' may be undefined, or defined from star imports: ctypes